src/home/NotificationCenter.tsx

```diff
@@ -19,9 +24,114 @@ import useSelector from 'src/redux/useSelector'
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'NotificationCenter'
 
 type NotificationsProps = NativeStackScreenProps<StackParamList, Screens.NotificationCenter>
 
+function useCleverTapNotifications() {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+
+  const messages = useSelector(cleverTapInboxMessagesSelector)
+
+  const notifications: Notification[] = []
+
+  if (messages) {
```

Comment:
my intention was to have some guard rails in case CleverTap sends us something invalid.
i moved that logic to the selector, and now it checks that it's indeed an array (`Array.isArray()`)